apps/cms.c

```diff
@@ -603,6 +603,20 @@ int cms_main(int argc, char **argv)
         goto opthelp;
     }
 
+    if (flags & CMS_CADES) {
+        if (flags & CMS_NOATTR) {
+            BIO_puts(bio_err, "Incompatible options: "
```

Comment:
Add a test if possible for this path

Reply:
a run(app()) in 80-cms_test.t should be fairly easy to add (doesnt need to be in a array)

Reply:
Thanks for the inspiration.
Tests added.